src/index.ts

```diff
@@ -58,9 +64,13 @@ export class AppInsightsTransport extends TransportStream {
       .setAutoCollectExceptions(false)
       .setAutoCollectRequests(false)
       .setAutoDependencyCorrelation(false) // this is needed because otherwise winston + AI + pm2 crashes for unknown reasons...
+      .setUseDiskRetryCaching(clientOptions ? clientOptions.useDiskRetryCaching : true)
```

Comment:
Is this wanted? Right now the value space is `true`, `false`, `undefined`. I guess it should just be `true` and `false`.

I guess if `useDiskRetryCaching` is undefined we want to fall back to `true`, right?